backend/tracim_backend/models/data.py

```diff
@@ -1173,6 +1173,18 @@ def children(self) -> ["Content"]:
 
         return valid_children
 
+    @property
+    def recursive_children(self) -> ["Content"]:
```

Comment:
use a method here, not a property, I think. `get_children_recursively()`

Reply:
done